calico/felix/devices.py

```diff
@@ -299,6 +299,8 @@ def remove_conntrack_flows(ip_addresses, ip_version):
 RTM_DELLINK = 17
 
 IFLA_IFNAME = 3
+IFLA_OPERSTATE = 16
+IF_OPER_UP = 6
```

Comment:
Why is up == 6?  Just wanted to check that it's not a bit field that we should be checking against a mask.